public/static/js/spawned_hosts.js

```diff
@@ -176,13 +176,9 @@ mciModule.controller('SpawnedHostsCtrl', ['$scope', '$window', '$timeout', '$q',
     }
 
     $scope.availableUnexpirableHosts = function () {
-      var available = $scope.maxUnexpirableHostsPerUser;
```

Comment:
Is this just better by convention, or do you have evidence that the other layout didn't work?

Reply:
The other one generated console errors since the for...of loop structure only works on objects, not arrays. Instead of fixing just that part, I rewrote it to be a bit shorter